File: compress_logo.js
const fs = require('fs');
const path = require('path');

const logoPath = path.join(__dirname, 'logo.png');
if (!fs.existsSync(logoPath)) {
    console.error("Error: logo.png not found");
    process.exit(1);
}

const fileBuffer = fs.readFileSync(logoPath);
const base64Data = fileBuffer.toString('base64');
const dataUrl = `data:image/png;base64,${base64Data}`;

fs.writeFileSync(path.join(__dirname, 'logo_base64.txt'), dataUrl);
console.log("Success! Base64 logo size:", dataUrl.length, "bytes");

File: compress_logo.py
import base64
import os
import sys

def compress_and_encode():
    workspace_logo = 'logo.png'
    if not os.path.exists(workspace_logo):
        print("Error: logo.png not found")
        sys.exit(1)
        
    # Read raw logo bytes
    with open(workspace_logo, 'rb') as f:
        img_bytes = f.read()
        
    # Convert to Base64
    base64_encoded = base64.b64encode(img_bytes).decode('utf-8')
    data_url = f"data:image/png;base64,{base64_encoded}"
    
    # Save to file
    output_path = 'logo_base64.txt'
    with open(output_path, 'w') as f:
        f.write(data_url)
    print("Success! Base64 logo size:", len(data_url), "bytes")

if __name__ == '__main__':
    compress_and_encode()
